merkle-distributor/distributor.ts

```diff
@@ -0,0 +1,23 @@
+import BalanceTree from "./balance-tree";
```

Comment:
What is this file?

Reply:
This file provides a Merkle Tree wrapper to build the customized Merkle Tree that we needs from runtime.